src/styles/product-card.scss

```diff
@@ -0,0 +1,96 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: $basicWhite;
+  border: 1px solid $lightgrey;
+  border-radius: 5px;
+
+  transition-duration: 300ms;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 5px;
+    color: $mainAccent;
+  }
+
+  &__code {
+    font-size: 10px;
+    align-self: start;
+    margin-bottom: 17px;
+    color: $secondaryColour;
+  }
+
+  &__rate {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $mainAccent;
+  }
+
+  &__price {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 15px;
+
+    &-label {
+      color: $secondaryColour;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      color: $mainAccent;
+    }
+  }
+
+  &__button {
+    width: 100%;
+    height: 40px;
+    border: 1px solid $accentColour;
+    border-radius: 5px;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-transform: uppercase;
+    color: $basicWhite;
+    background-color: $accentColour;
+    transition-duration: 300ms;
+
+    &:hover {
+      background-color: $basicWhite;
+      color: $accentColour;
+    }
+  }
+
+  &:hover {
+    transform: scale(120%);
```

Comment:
The scale value in the transform property should be a decimal value. Instead of 'scale(120%)', it should be 'scale(1.2)'.